packages/swirl-tokens/scripts/build.ts

```diff
@@ -24,14 +24,9 @@ StyleDictionary.registerFormat({
         }
 
         if (theme === "light") {
```

Comment:
**issue:** The if statement seems obsolete.

Reply:
yeah it is now